GitUI/CommandsDialogs/SettingsDialog/MergeToolsHelper.cs

```diff
@@ -402,8 +359,28 @@ private static string GetRegistryValue(RegistryKey root, string subkey, string k
             return value ?? string.Empty;
         }
 
-        private static string GetVisualStudioPath()
+        private static string GetVsDiffMergePath()
         {
+            //%ProgramFiles(x86)%\Microsoft Visual Studio\Installer\vswhere.exe
+            var vsVersionNumber = new string[] { "2020", "2019", "2018", "2017" };
+            var vsVersionType = new string[] { "Professional", "Community" };
```

Comment:
@RussKie 

>If you fix a bug - just add a PR for the bug, if you add a feature - do a PR for the feature...

Let me laugh, with your great bible rules! It's what I do most of the time but when the code is intertwined with what you start to do or that is a little fix that will be a real pain to fix in it's how branch, create a PR based on the first PR,........

I can't know that my code will be reviewed by a show-off  that think he know better than everyone without understanding how the code was before, didn't know what are the capacities (mostly in term of time) that the dev can give to the project, .... that require some 

>The goal is should be improving UX and leaving the codebase in a better state than you found it.

That's what I did (talk to my to all the deduplication of code I made! And this fix a bug) . But I can remove my fix and let you fix it or I can wait your commit that I will add to this PR. I won't spend more time on this part of the code anymore, I have no time (I would like to finished that before taking a month away...).

Things aside: I was happy to contribute from time to time to this project (even if I struggle to find time), but I should say that you are making this a pain since you came in the project. I see all the efforts you put on the project. Admit that not everyone could put the same amount of time or energy. I left the code a little better than what I found and I'm not able to pass more time to make it as perfect as YOU want (on something I was not expectiong to work on!)

Sorry but sometimes you should understand other and that your point of view is ALWAYS the better solution. I had to make tradeoff (everyone have to), I did them **carefully**